edx_event_bus_kafka/tests/test_config.py

```diff
@@ -0,0 +1,50 @@
+"""
+Test common configuration loading.
+"""
+
+from unittest import TestCase
+
+from django.test.utils import override_settings
+
+from edx_event_bus_kafka import config
+
+try:
+    from confluent_kafka.schema_registry import SchemaRegistryClient
+except ImportError:  # pragma: no cover
+    SchemaRegistryClient = None  # pragma: no cover
```

Comment:
See https://github.com/openedx/event-bus-kafka/blob/main/docs/decisions/0005-optional-import-of-confluent-kafka.rst. Let's use `confluent_kafka`.

Reply:
The tests are always run with `confluent_kafka` installed, so there's nothing that would check `if confluent_kafka`. This is just to make the module importable. I could probably even replace `SchemaRegistryClient = None` with `pass`.

Reply:
1. Curious where and how does this breaks if you don't have a try block. Would edx-platform complain about test code in this dependency?
2. If this is needed, maybe something like `confluent_kafka = None` accompanied by an assertion in the test that `confluent_kafka is not None` would make it more clear that tests can't run without this.

Reply:
What I picture is that there _might_ be some code in edx-platform that tries loading all modules... or something. I don't know. It seems safe to have the try-block there.

I don't think it's worth building up test code around it, though. Tests will fail regardless -- especially if I change it to a `pass`.

Reply:
From the perspective of someone trying to understand the tests by looking at the code, I found this confusing, and am just trying to find a way to make it clear (via comment, assertion, or otherwise).

Another proposal is to add a comment like the following:
```
    # Although confluent_kafka is required for tests to pass, adding protective code
    # just in case the module is imported elsewhere.
    # See https://github.com/openedx/event-bus-kafka/blob/main/docs/decisions/0005-optional-import-of-confluent-kafka.rst
    pass
```

Reply:
Can you take a look at https://github.com/openedx/event-bus-kafka/pull/30? I've made it a separate PR.

Reply:
Thanks. Looks good, once the other PR is merged.